cosmos/dbt/graph.py

```diff
@@ -148,35 +230,30 @@ def load_via_dbt_ls(self) -> None:
             self.project.project_name,
             self.project.dbt_project_path,
         )
+        if self.project.dbt_project_path is None:
+            raise CosmosLoadDbtException("Unable to dbt ls load a  project without a project path.")
```

Comment:
This was needed or else I was getting typing errors for `DbtGraph.project.dbt_project_path` which could be None

Reply:
This will probably be changed as part of #634, by @MrBones757